_data/sections.yml

```diff
@@ -66,6 +66,10 @@
   title: Identity Management
   icon: user
 
+- id: insurance
+  title: Insurance
+  icon: protect
```

Comment:
Is this a lock icon, the same as for Security category?
Is there something else we can use more representative for "insurance? 🤔

Reply:
It's a "shield" icon which I found suitable.
https://semantic-ui.com/elements/icon.html

Reply:
I agree with what @mxxcon said. It could be confused with security.
I also feel that a shield doesn't signal insurance as much as it does protection (such as law enforcement).

What about the `life ring` icon?
To me that signals insurance however that might be a cultural thing.
Appreciate feedback! 😄